docs/language/crypto.mdx

```diff
@@ -107,11 +127,23 @@ struct PublicKey {
     ): Bool
 
     /// Verifies the proof of possession of the private key.
-    /// This function is only implemented if the signature algorithm of the public key is BLS, it errors if called with any other signature algorithm. 
+    /// This function is only implemented if the signature algorithm of the public key is BLS (BLS_BLS12_381).
+    /// It errors if called with any other signature algorithm. 
```

Comment:
```suggestion
    /// If called with any other signature algorithm, the program aborts 
```

Reply:
Also, please update the docstring in the code, here: https://github.com/onflow/cadence/blob/master/runtime/sema/type.go#L6260